ReleaseNotes.md

```diff
@@ -40,6 +40,10 @@ Git for Windows also contains Embedded CAcert Root Certificates. For more inform
 
 This package contains software from a number of other projects including Bash, zlib, curl, tcl/tk, perl, MSYS2 and a number of libraries and utilities from the GNU project, licensed under the GNU General Public License. Likewise, it contains Perl which is dual licensed under the GNU General Public License and the Artistic License.
 
+## Upcoming breaking changes
+
+We plan to update the included bash to version 5.1 soon after Git for Windows 2.35.0 is released. Please check your shell scripts for potential compatibility issues.
+
```

Comment:
If we move it into the "Changes since..." section, it will be part of the announcement mail...

Reply:
That's not the worst idea I've ever heard. Give me a few minutes to force push an update.

Reply:
Before hitting the merge button, one last idea: maybe mention that we're on Bash 4.4 right now?

Reply:
That probably wouldn't hurt. I've added it.